tests/test_smoketest.py

```diff
@@ -12,9 +12,26 @@
 import cohortextractor
 
 
+@pytest.fixture
+def study(request):
+    study_name = request.function.__name__
+    study_path = os.path.join(os.path.dirname(__file__), "studies", study_name)
```

Comment:
Bikesheddy, but I'd be inclined to keep the name `fixtures` in the path, given that immediately tells you that the directory contains test fixtures, rather than test code. I think I'd be happy enough with `study_fixtures` or something like that if you felt that was useful.

Renaming to `test_smoketest` to match the test file name is a good call though.

Reply:
Yes. I wasn't happy with this. I preferred having "fixtures" in the path, but I wanted to make it clear that the contents of each of those directories was a study. I think maybe I like `fixtures/studies/test_smoketest` best now. I rejected that before because the directory with only one child seems redundant, but I think it captures the semantics best (and directories are just an illusion anyway...).

I'll change to that, but please object if you don't like it.

Reply:
No, that sounds good. I share your instinctive aversion to superfluous nesting, but I agree that the clarity is worth the (tiny) cost.